test/integration/IntegrationTests.spec.ts

```diff
@@ -369,6 +369,60 @@ describe("Upload DAR integration test", () => {
     });
 });
 
+describe('Template identifier retrieval', () => {
+
+    function getTemplateIds(archivePayload: string): string[] {
+        const templateNames: string[] = [];
+        const archive = lf.ArchivePayload.deserializeBinary(archivePayload as unknown as Uint8Array).getDamlLf1()!;
+        for (const damlModule of archive.getModulesList()) {
+            if (damlModule.hasNameDname()) {
+                expect(damlModule.hasNameDname()).to.be.true;
```

Comment:
is this assert necessary, since it is anyway guarded by the `if` clause before it?

Reply:
It's not, the idea was to expect that everything else was interned if the first check says that the module name is interned. Fixing, thanks.

Reply:
Addressed by https://github.com/digital-asset/daml-js/pull/104/commits/69cb1d5275bef91535b977ca61d2bfaf378fc55f